src/lib.rs

```diff
@@ -349,7 +349,7 @@ pub fn full_roots(i: usize, nodes: &mut Vec<usize>) {
 }
 
 #[inline]
-fn is_even(num: usize) -> bool {
+pub(crate) fn is_even(num: usize) -> bool {
   (num & 1) == 0
```

Comment:
Aaah nice one. Didn't know you could do this. I guess exposing the function but only within the crate?

Reply:
exactly!